kolibri/core/assets/src/vue/assessment-wrapper/index.vue

```diff
@@ -0,0 +1,92 @@
+<!--
+This template is intended to act as the default wrapper for assessment focused rendering
+plugins.
+
+As such, it provides display of current mastery progress, and manages all mastery/attempt log
+oriented data synchronization.
+-->
```

Comment:
It operates in concert with the content-renderer. At the moment, it has to be specifically invoked inside the assessment renderer, and then all content is put inside the slot.

Assessment specific data management is then done in this component too.